.asciidoctor/lib/const-inline-macro/extension.rb

```diff
@@ -49,8 +49,8 @@ def process parent, target, attrs
       end
     end
     if const_value[0][0].nil?
```

Comment:
This will again fail. Because `const_value` will be null as it won't find line with require const_name
Look at [1](https://github.com/arquillian/smart-testing/blob/master/.asciidoctor/lib/const-inline-macro/extension.rb#L29) & [2](https://github.com/arquillian/smart-testing/blob/master/.asciidoctor/lib/const-inline-macro/extension.rb#L42)

Reply:
Ah right :) I just assumed it's initialized as `[]`

Reply:
I think we should check for both, really `const_value.nil? or const_value[0][0].nil?`